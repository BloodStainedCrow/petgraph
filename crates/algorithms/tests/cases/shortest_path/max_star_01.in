version https://git-lfs.github.com/spec/v1
oid sha256:36bdce1c462760d4c4f5c5e76c42bf06d6f134b71bf34eb39c7b994596ce55bc
size 11777953
