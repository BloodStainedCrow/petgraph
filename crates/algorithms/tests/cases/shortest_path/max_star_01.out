version https://git-lfs.github.com/spec/v1
oid sha256:9a7b41bbf2d24cd5ce10a32a675acf937cf3df56776d13dd075fc3df252024ae
size 40
